2018-03-28T06:34:27.802Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))
    at /home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-28T06:34:27.803Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-03-28T06:34:27.803Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
2018-03-28T06:52:02.686Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)
    at /home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-28T06:52:02.703Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-03-28T06:59:49.312Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)
    at /home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-28T06:59:49.312Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-03-28T07:12:48.818Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)
    at /home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-28T07:12:48.818Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-03-28T07:15:51.965Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)
    at /home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-28T07:15:51.966Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/malak/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
